controller/src/main/java/io/pravega/controller/server/Main.java

```diff
@@ -91,6 +91,7 @@ public static void main(String[] args) {
                     .host(Config.REST_SERVER_IP)
                     .port(Config.REST_SERVER_PORT)
                     .tlsEnabled(Config.TLS_ENABLED)
+                    .tlsProtocolVersion(Config.TLS_PROTOCOL_VERSION.toArray(new String[Config.TLS_PROTOCOL_VERSION.size()]))
```

Comment:
This is an example where you convert a List to an array. We could have just passed in the List itself here and hold it as a list inside the class.

Reply:
Passing List to end components  again requires converting it to String[], for example Grizzly's library used in RESTServer takes only String[] as parameter.